lib/ansible/modules/database/influxdb/influxdb_retention_policy.py

```diff
@@ -123,7 +143,9 @@ def influxdb_argument_spec():
         port=dict(default=8086, type='int'),
         username=dict(default='root', type='str'),
         password=dict(default='root', type='str', no_log=True),
-        database_name=dict(required=True, type='str')
+        database_name=dict(required=True, type='str'),
+        ssl=dict(required=False, type="bool", default=False),
+        verify_ssl=dict(required=False, type="bool", default=False)
```

Comment:
For convention, please rename this to `validate_certs` or add an alias e.g. `aliases=['validate_certs']` to it (dont' forget to add the aliases also to the docs section.
The default should be `True`.